tests/quality_pragmaitc/test_refinement.py

```diff
@@ -2,20 +2,25 @@
 from goald.quality.pragmatic.model.task import Task
 from goald.quality.pragmatic.model.delegation import Delegation
 from goald.quality.pragmatic.model.context import Context
-from goald.quality.pragmatic.model.metric import CommonMetrics
+from goald.quality.pragmatic.model.common_metrics import CommonMetrics
+from goald.quality.pragmatic.model.goal import Goal
+from goald.quality.pragmatic.model.interpretation import Interpretation
+from goald.quality.pragmatic.model.quality_constraint import QualityConstraint
+
 
 def test_refinement():
     refinement = Refinement()
     task = Task()
     delegation = Delegation()
-    #goal = Goal()
+    goal = Goal()
+
+    assert task.myType() is refinement.TASK
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4861978)